### ISSUE TITLE
Ability to add the message timestamp as a column to the main pane

### ISSUE THREAD

**Issue by l0psec** [redcanaryco/mac-monitor#14]
**At a high level -- can you summarize your request?**
Would like the ability to see time stamps for all events in the main pane without having to go into the event metadata for each event. 

**What is the current alternative solution?**
Timestamps do not appear to be visible in the main page. Going to the event metadata allows you to see the timestamps.

**Comment by Brandon7CC** [comment]
Done and done! See `v1.0.3` thank you for the awesome feature request @l0psec!
